src/ui/SortingButton/SortingButton.jsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from 'react';
+import './SortingButton.css';
+import { SHARE_BUTTON, RENAME_BUTTON, DELETE_BUTTON } from './constant';
+import { useGetUserFolders } from '../../data-access/useGetUserFolders';
+
+const SortingButton = () => {
+  const [activeButton, setActiveButton] = useState(0);
```

Comment:
```suggestion
  const [activeButtonIdx, setActiveButtonIdx] = useState(0);
```
Index 임을 나타내주시면 좋을 거 같아요!